external/solr/src/test/java/org/apache/stormcrawler/solr/persistence/IndexerBoltTest.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Licensed to DigitalPebble Ltd under one or more contributor license agreements. See the NOTICE
```

Comment:
This is not the right header I'm afraid.  It should not contain any reference to DigitalPebble at all.
Please have a look at the Java classes in the main branch

Reply:
Sorry about that. I updated the headers - the headers in branch 851 seem to be the old ones.